src/bitcoind-res.rc

```diff
@@ -17,13 +17,13 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "Bitcoind (OSS daemon/client for Bitcoin)"
+            VALUE "FileDescription",    "bitcoind (Bitcoin client as a JSON-RPC server)"
```

Comment:
I like "headless Bitcoin node". "Client" is also the wrong word to use in a P2P world.